### PR TITLE
Use New Sample App And Fix Exporter Endpoint

### PR DESCRIPTION
**Pull request by sethAmazon**
**Description:** Use New Sample App And Fix Exporter Endpoint
Ex. Adding a feature - Explain what this achieves.> Updating the sample app

**Testing:** Docker compose up -d then tailing the logs to look for errors

### CODE REVIEW

File: examples/docker/docker-compose.yaml [hunk 1]
```diff
@@ -33,9 +33,9 @@ services:
   # Sample web application which will generate Metrics and Traces data if the enable API is called
   # src - https://github.com/aws-observability/aws-otel-test-framework/tree/terraform/sample-apps/spark
   ot-sample-app:
-    image: aottestbed/aws-otel-collector-java-sample-app:0.6.0
+    image: public.ecr.aws/a3i2u7g3/aws-otel-test-framework-sample-apps-spark:1.4.1
```
Comment (sethAmazon) [inline examples/docker/docker-compose.yaml:36]: This needs to be changed from my public ecr repo to a public ecr repo owned by agents aws account.

Comment (sethAmazon) [reply examples/docker/docker-compose.yaml:36]: It seems this is using the docker hub link  https://hub.docker.com/layers/aottestbed/aws-otel-collector-java-sample-app/0.6.0/im[…]cd286de06a31a43da9765ef679fabe04989820e83a3a2?context=explore.

Comment (sethAmazon) [reply examples/docker/docker-compose.yaml]: Changed version to 0.10.0

File: examples/docker/docker-compose.yaml [hunk 2]
```diff
@@ -33,7 +33,7 @@ services:
   # Sample web application which will generate Metrics and Traces data if the enable API is called
   # src - https://github.com/aws-observability/aws-otel-test-framework/tree/terraform/sample-apps/spark
   ot-sample-app:
-    image: aottestbed/aws-otel-collector-java-sample-app:0.9.0
+    image: public.ecr.aws/a3i2u7g3/aws-otel-test-framework-sample-apps-spark:0.10.0
```
Comment (mxiamxia) [inline examples/docker/docker-compose.yaml:36]: have you made any changes to the sample app code or just use the current code and republish it to ECR?

**Review by mxiamxia — APPROVED** [review]

File: examples/docker/docker-compose.yaml (hunk 2 above)
Comment (sethAmazon) [reply examples/docker/docker-compose.yaml:36]: Yes the test changes should be in the new ecr

Comment (sethAmazon) [reply examples/docker/docker-compose.yaml:36]: RUN wget https://github.com/aws-observability/aws-otel-java-instrumentation/releases/download/v1.4.1/aws-opentelemetry-agent.jar

Comment (jefchien) [reply examples/docker/docker-compose.yaml]: The changes should be the same as the ones in this PR: https://github.com/aws-observability/aws-otel-test-framework/pull/321

**Review by mxiamxia — APPROVED** [review]

**Pull request merged**